pyk/src/pyk/kast/outer_lexer.py

```diff
@@ -190,23 +191,29 @@ class State(Enum):
 
 
 class LocationIterator(Iterator[str]):
-    _line: int = 1
-    _col: int = 0
+    _line: int
+    _col: int
     _iter: Iterator[str]
+    _nextline: bool
 
-    def __init__(self, x: Iterable[str]) -> None:
+    def __init__(self, x: Iterable[str], line: int = 1, col: int = 0) -> None:
```

Comment:
Why the discrepancy in default argument values?

Reply:
The column is incremented when the iterator returns a character. So when it returns the first character, the location will be at column 1, which is where it's supposed to be.